test/form/govuk-notify.test.js

```diff
@@ -94,6 +105,12 @@ privateLimitedCompany
 \`\`\`
 
 
+## Autocomplete field
+\`\`\`
+910400044
+\`\`\`
```

Comment:
Unlike checkboxes and radios etc, the raw value `910400044` is output not "Czech Republic"

Should we change this?

Reply:
I would prefer the human readable label personally. I've flagged it with Naz.